.github/workflows/cspell-cli.yml

```diff
@@ -41,5 +41,4 @@ jobs:
             cspell-cache-v2-${{ runner.os }}-
 
       - name: cspell@latest
-        run: npx cspell@latest --cache --cache-strategy=content --cache-location=.cspellcache --exclude="yarn2" --no-progress "**"
-        # run: npx cspell@latest --cache --cache-strategy=content --cache-location=.cspellcache --exclude="yarn2" "**"
```

Comment:
This comment originates from https://github.com/streetsidesoftware/cspell/pull/2303. Looks like a leftover from some experiment. Removing to avoid confusion.